.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+# Pull Request Process
+
+Please read
+[Style Guide Review Process](https://wiki.corp.mongodb.com/display/DE/Technical+Writer+Style+Guide+Review+Process)
+
+Contributors should take the following actions:
+
+- Request a PR review in #docs-style-guide Slack channel.
+- Wait for review approvals from all the Style Guide team reps. Once all 
+  the team reps have approved the PR, they will merge the PR and notify you
+  of the merge in the Slack channel.
+
+# Pull Request Description
+
+JIRA URL: <https://jira.mongodb.org/browse/DOCSP-NNNNN>
```

Comment:
[question]

Do we want to add a section for the writer to add a description in prose? On server docs, we've found it useful to allow writers to provide additional context / notes for review if it would be helpful.

See my suggestion for what this might look like if added to the template.

```suggestion
# Pull Request Description

<Summary of changes (Optional)>

JIRA URL: <https://jira.mongodb.org/browse/DOCSP-NNNNN>
```